robottelo/datafactory.py

```diff
@@ -33,12 +33,11 @@ def func_wrapper(*args, **kwargs):
             # New UI tests are written using pytest, update dict to support
             # pytest's parametrize
             if 'ui' in args or kwargs.get('interface') == 'ui':
-                # fixme: Chromedriver only supports BMP chars, but fauxfactory
-                # generates both BMP and SMP chars. Disabling all affected
-                # string types until resolved
+                # Chromedriver only supports BMP chars
                 if settings.webdriver == 'chrome':
-                    for unsupported_type in ('cjk', 'latin1', 'utf8', 'html'):
-                        dataset.pop(unsupported_type, None)
+                    utf8 = dataset.pop('utf8', None)
+                    if utf8:
```

Comment:
could you please explain that logic? I mean, why utf8 is so special that it is generated explicitly?

Reply:
As the comment says - chromedriver supports typing chars from Basic Multilingual Plane only, and rises exceptions on attempt to enter smth from Supplementary Multilingual Plane. Pls take a look at https://en.wikipedia.org/wiki/Plane_(Unicode)
All the datasets like `cjk`, `latin1` are inside BMP so they are supported, but `gen_utf8` returns chars from BMP and SMP by default, and, as of latest fauxfactory 3.0.2 we have to call `gen_utf8` with `smp=False` arg not to generate SMP for chrome.